packages/connect-web/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@trezor/connect-web",
-    "version": "9.0.11",
+    "version": "9.1.0",
```

Comment:
Why it did not created CHANGELOG for `connect-web`? 
We have definitely make changes there and if I run the git command manually it gives me commits since last release:

```
git log --oneline --max-count 1000 --pretty=tformat:"-   %s (%h)" -- ./packages/connect-web
-   fix(connect-web): closing popup when in error and other request (e5e771fd42)
-   fix(connect-web): separate close with clearing popup (eb8255ae45)
-   chore(deps): update (a21a081ba1)
-   feat(connect-web): not clossing popup when debug mode (3d9fa1e719)
-   chore(connect): fix extraneous dependencies (6707f7df93)
-   chore(transport): refactor (f7b97fb681)
-   test(connect-web): remove unused param (99d1f9847b)
-   fix(connect-web): playwright test script (7725e530a8)
-   npm-release: @trezor/connect 9.0.11 (9a608986af)

```

Reply:
version should be the same, thats right!